### ISSUE TITLE
Plugins error (even though not loading any plugins)

### ISSUE THREAD

**Issue by resthedev** [atomiks/tippyjs-react#379]
I am getting the following error:

```
2961-809c1f3bec2099a2.js:1 TypeError: e.fn is not a function
    at 4196-45a1ea360224e18c.js:1:31229
    at Array.map (<anonymous>)
    at 4196-45a1ea360224e18c.js:1:31204
    at 4196-45a1ea360224e18c.js:1:37405
    at Array.reduce (<anonymous>)
    at eZ (4196-45a1ea360224e18c.js:1:27992)
    at 4196-45a1ea360224e18c.js:1:1643
    at ap (bce60fc1-0399b9eba6931719.js:9:68389)
    at ah (bce60fc1-0399b9eba6931719.js:9:68449)
    at ay (bce60fc1-0399b9eba6931719.js:9:68888)
```

Looking into the tippy.js repo code, I saw that [this line](https://github.com/atomiks/tippyjs/blob/ad85f6feb79cf6c5853c43bf1b2a50c4fa98e7a1/src/createTippy.ts#L136C1-L137C1) has to do with plugins. I am not loading any plugins with tippy, so I'm not sure why it is trying to call fn() on a plugin. I saw that the `@tippyjs-react` library does load some of its own plugins, so maybe there is an issue there?

I wasn't having this issue prior, but started seeing it pop up now after I upgraded Next version 13.4.9 -> 13.4.10.

**Comment by CopyJosh** [comment]
Same issue. Not using the new app directory, so not sure what the fallout is from this last upgrade...

Been meaning to drop this dead package for awhile now... guess that time has come.

**Comment by atomiks** [comment]
Likely a Next.js minification bug, was also filed here: https://github.com/atomiks/tippyjs/discussions/1132

**Comment by DaanWevolver** [comment]
I have exactly the same issue after upgrading to next 13.4.11 (from next 12). downgrading to 13.4.9 fixed the issue for me.

**Comment by icyJoseph** [comment]
Hi, I believe the problem might be SWC. I've set up a [playground example here](https://play.swc.rs/?version=1.3.71&code=H4sIAAAAAAAAA02OvQrDMAyEdz%2BFRgdCSn%2Bm0GTrXrqGDK0jpwbXCrZcCiXvXjvpEE0fd8fpFLnAMNk4GgcNfAUA01TDvkxkUXMNh4zejM%2FEx8wPYqZXDadSzELgZyLPoKNTbMjBlYLJcEM3oJfF0gkx4EVrVCyT0rSrmE5t%2Foc0oFuxr5Kh7iy7vtgkyWJlaZT%2F%2FGLNhcj7kKN3cB7MG3Zt3iV%2BhE9Hk9sAAAA%3D&config=H4sIAAAAAAAAA32VwXLjIAyG73mKjM%2B99rIPsLc%2BA0OMcMhi5EEijaeTd19BnNRt5Fw89v8hCSQhf%2B32%2B%2B5Effdn%2FyWv8jHZTJAf36LQnNheROl4noD6HCbu3u6UqSLOBZpyvYGOs03kMY9rTxlsz3dhEa8%2FzWweoC7pgN6XIF1EJFiCLNIYUvDz2neP45SBaKWJKu7KCImr7G0keFuzjJ%2F0w2%2FTD4gRZPfbxFgyITEMkDXHPcZoJwJztlnxUndqcyDUQlRYGJyZMk4qTy5wwCQxn6kD60yPDhQUMvQczqCZSSwxSyTHU87TsINDGYbWGb%2Bs4WxjsazEhEsriexW8XrEQGx8SVoKb3AjBze4JPe3ZfAmA5ecnu1OGNJGTf4BSAaiJUp2BM1vW%2BGln7as%2FUvLkLy0LM8Kl%2FbWTplgkKSaELyS2ZoZyBy0amZwpYea2V7bzoI30kfBgQHvpVcU1%2FQZuD9qQetoQK8Aqa%2F1WlfdgHncwg1eL8QL%2FFdOyXqDLStGy8dtSvN4wPgiwAh8RPdigZSCcRtnmRKXaZuX5EBaA5y6pFADz0NALgCjiW1WPvWGXA%2FxaIaIh%2B8x8Txv26ML9IGuxHWTdeNdWX4Ktbp1KEsxRkwn6h7Wj0Hc9r677v4DNkYPd1IGAAA%3D)


I've tested in Next.js and running with `swcMinify` set to false works. So I guess that's a pretty good indicator anyway. 

- https://nextjs.org/docs/architecture/nextjs-compiler#minification

For completeness sake, what happens is that:

- https://github.com/atomiks/tippyjs-react/blob/master/src/className-plugin.js

Gets minified into an empty object, and calling `fn()` on it, crashes out.

**Comment by aslakson** [comment]
I ran into this issue and forked this repo with a temporary fix. I am filtering the plugins list and hard-coded the classNamePlugin in the two places it was being imported as a temporary workaround. I plan on switching back to this repo as soon as SWC stops doing what it's doing. I have no plans to support other bug fixes or changes, but use it if you want:

[https://github.com/aslakson/tippyjs-react](https://github.com/aslakson/tippyjs-react)

```
yarn add @aslakson/tippyjs-react
```

**Comment by FengeRVictor** [comment]
Same issue for me started with nextjs 13.4.10

**Comment by icyJoseph** [comment]
I'll be fixed by https://github.com/swc-project/swc/pull/7702 as indicated here: https://github.com/vercel/next.js/issues/53151#issuecomment-1655734703

**Comment by resthedev** [comment]
This appears fixed for me now as of NextJS v13.4.15

Thank you team!
